quickstart/docker-compose.yml

```diff
@@ -0,0 +1,143 @@
+version: '3'
+
+services:
+  # ===========================================================================
+  # sso services
+  #
+  # Here we provide a minimal sso installation for demo purposes that allows
+  # any valid google account to log in, and hard codes the various secrets
+  # required to secure communication.
+  #
+  # A more realistic deployment will likely require a specific organization's
+  # email domain and restrict access to upstream services based on Google Group
+  # membership.
+  #
+  # The sso-proxy service is handing requests to any domain under
+  # *.sso.localtest.me and the sso-auth service is available at sso-
+  # auth.localtest.me.
+  #
+  # There are two upstream services defined below, which can be accessed at
+  #  - http://hello-world.sso.localtest.me
+  #  - http://httpbin.sso.localtest.me
+  # ===========================================================================
+  sso-proxy:
+    image: buzzfeed/sso:latest # change this to `build: ..` to try local changes
+    entrypoint: /bin/sso-proxy
```

Comment:
do we need `working_dir: /go/src/github.com/buzzfeed/sso/` here to handle the static css properly?

i get this fail without it

![screen shot 2018-08-23 at 10 35 06 am](https://user-images.githubusercontent.com/5041307/44532326-de66a280-a6c0-11e8-8718-821fb8c5ba23.png)

Reply:
@danbf I'm pretty sure this is an indication that you're not actually running the latest docker image

Reply:
nvm, yup: this fixed it `docker rm -f $(docker ps -aq); docker rmi -f $(docker image list -q)`